Framework/DataHandling/src/LoadILLIndirect2.cpp

```diff
@@ -80,6 +80,9 @@ void LoadILLIndirect2::init() {
   declareProperty(std::make_unique<WorkspaceProperty<>>("OutputWorkspace", "",
                                                         Direction::Output),
                   "The name to use for the output workspace");
+  declareProperty(std::make_unique<PropertyWithValue<bool>>(
```

Comment:
Just an early comment, generally bool properties should be avoided. String property with array validator is more scalable in case there could be other options to be added in future. In this case: `LoadDetectors`=['Spectrometer', 'Diffractometer'] is the way to go.